config/initializers/event_bus.rb

```diff
@@ -54,6 +72,14 @@
     DiscussionReader.for_model(model, actor).set_volume_as_required!
   end
 
+  config.listen('discussion_reader_viewed!') do |discussion, discussion_reader, actor|
```

Comment:
Do you think we could take the `discussion_reader` param out of this, since we don't use it, and just do `|discussion, actor|`?